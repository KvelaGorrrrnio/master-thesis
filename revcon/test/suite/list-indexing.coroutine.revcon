@coroutine

coroutine main(int x) {
  local list int a [1,2,3,4,5]
  x += a[1+2]
  x *= a[4/2]
  delocal list int a [1,2,3,4,5]
}
