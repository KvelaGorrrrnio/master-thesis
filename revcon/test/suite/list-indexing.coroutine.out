test/suite/list-indexing.coroutine.revcon:
[store]
  main: Ã˜
  x: 12

[stack]
[spawn main main begin () | resume main (x) | despawn main main end ()] <|> []

